decidim-budgets/lib/decidim/budgets/feature.rb

```diff
@@ -22,11 +22,11 @@
   feature.settings(:global) do |settings|
     settings.attribute :total_budget, type: :integer, default: 100_000_000
     settings.attribute :vote_threshold_percent, type: :integer, default: 70
-    settings.attribute :comments_always_enabled, type: :boolean, default: true
+    settings.attribute :comments_enabled, type: :boolean, default: true
```

Comment:
@oriolgual If the setting is not found it uses the default value so the migration is not needed.